Resources/Prototypes/_RMC14/Entities/Mobs/Species/vulpkanin.yml

```diff
@@ -0,0 +1,101 @@
+- type: entity
+  save: false
+  parent: CMBaseMobSpeciesOrganic
+  id: CMMobVulpkanin
```

Comment:
New content IDs should be RMC and not CM

Reply:
slightly confused on this, I can change it but I didn't see Avali or Felinids having that on their IDs?

Reply:
Just removing the CM part all together is fine too

Reply:
ID has been changed to RMCMobVulpkanin